### PR TITLE
Hindi Translation done

### PR DESCRIPTION
**Pull request by RushaliPadhiary**
It has the hindi translation for the Wallet app

### CODE REVIEW

File: locale/hi/translation.toml [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+send = "भेजें" # Send
+receive = "प्राप्त करें" # Receive
+contacts = "संपर्क" # Contacts
+name = "नाम" # Name
+username = "उपयोगकर्ता नाम" # Username
+addressOrXPub = "एड्रेस या XPub" # Address or XPub
+back = "बैक" # Back
+chooseAContact = "एक संपर्क चुनें" # Choose a Contact
+createContact = "संपर्क बनाएं" # Create Contact
+encryptFirstForContacts = "जब आप डैशबोर्ड में "{button}" पर क्लिक करते हैं, तो आप PIV प्राप्त करना आसान बनाने के लिए एक संपर्क बना सकते हैं!" # Once you hit \"{button}\" in the Dashboard, you can create a Contact to make receiving PIV easier!
```
Comment (Duddino) [inline locale/hi/translation.toml:81]:
```suggestion
encryptFirstForContacts = "जब आप डैशबोर्ड में \"{button}\" पर क्लिक करते हैं, तो आप PIV प्राप्त करना आसान बनाने के लिए एक संपर्क बना सकते हैं!" # Once you hit \"{button}\" in the Dashboard, you can create a Contact to make receiving PIV easier!
```

File: locale/hi/translation.toml [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+shareContactURL = "शेयर कांटेक्ट URL" # Share Contact URL
+setupYourContact = "अपने संपर्क को सेटअप करें" # Setup your Contact
+receiveWithContact = "सिंपल यूजरनेम-बेस्ड कांटेक्ट का उपयोग करके प्राप्त करें" # Receive using a simple username-based Contact
+onlyShareContactPrivately = "<b>केवल</b> अपने कांटेक्ट को विश्वसनीय लोगों (परिवार, दोस्तों) के साथ साझा करें" # <b>Only</b> share your Contact with trusted people (family, friends)
+changeTo = "चेंज टू" # Change to
+contact = "संपर्क" # Contact
+xpub = "XPub" # XPub
+addContactTitle = "{strName} को संपर्क में जोड़ें" # Add {strName} to Contacts
+addContactSubtext = "एक बार जोड़े जाने पर आप {strName} को उनके नाम से (टाइपिंग या क्लिकिंग के माध्यम से) ट्रांजैक्शन भेज सकेंगे, अब और एड्रेसिस की ज़रूरत नहीं, सब कुछ आसानी से।" # Once added you'll be able to send transactions to {strName} by their name (either typing, or clicking), no more addresses, nice 'n easy.
+addContactWarning = "सुनिश्चित करें कि यह असली "{strName}" है, अज्ञात स्रोतों से संपर्क अनुरोध स्वीकार न करें!" # Ensure that this is the real \"{strName}\", do not accept Contact requests from unknown sources!
```
Comment (Duddino) [inline locale/hi/translation.toml:91]:
```suggestion
addContactWarning = "सुनिश्चित करें कि यह असली \"{strName}\" है, अज्ञात स्रोतों से संपर्क अनुरोध स्वीकार न करें!" # Ensure that this is the real \"{strName}\", do not accept Contact requests from unknown sources!
```

File: locale/hi/translation.toml [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+editContactTitle = ""{strName}" संपर्क बदलें" # Change \"{strName}\" Contact
```
Comment (Duddino) [inline locale/hi/translation.toml:92]:
```suggestion
editContactTitle = "\"{strName}\" संपर्क बदलें" # Change \"{strName}\" Contact
```

File: locale/hi/translation.toml [hunk 4]
```diff
@@ -0,0 +1,335 @@
...
+popupWalletWipeNote = "यदि आपने अपने प्राइवेट की या सीड फ्रेज का बैकअप नहीं लिया है, तो आप अपने फंड्स तक पहुँच खो देंगे" # You will lose access to your funds if you haven't backed up your private key or seed phrase
+popupSeedPhraseBad = "अप्रत्याशित सीड फ्रेज" # Unexpected Seed Phrase
+popupSeedPhraseBadNote = "सीड फ्रेज या तो अमान्य है या MPW द्वारा उत्पन्न नहीं की गई थी।<br> क्या आप अभी भी आगे बढ़ना चाहते हैं?" # The seed phrase is either invalid or was not generated by MPW.<br>Do you still want to proceed?
+popupCreateProposal = "क्रिएट प्रपोजल" # Create Proposal
+popupCreateProposalCost = "कोस्ट" # Cost
+popupProposalName = "प्रपोजल नाम" # Proposal Name
+popupProposalAddress = "प्रपोजल ऐड्रेस (वैकल्पिक)" # Proposal Address (Optional)
+popupProposalDuration = "चक्रों में अवधि" # Duration in cycles
+popupProposalPerCycle = "प्रति चक्र" # per cycle
+popupProposalEncryptFirst = "आपको प्रपोजल बनाने से पहले "{button}" पर क्लिक करना होगा!" # You need to hit "{button}" before you can create proposals!
```
Comment (Duddino) [inline locale/hi/translation.toml:190]:
```suggestion
popupProposalEncryptFirst = "आपको प्रपोजल बनाने से पहले \"{button}\" पर क्लिक करना होगा!" # You need to hit "{button}" before you can create proposals!
```

File: locale/hi/translation.toml [hunk 5]
```diff
@@ -0,0 +1,335 @@
...
+MN_STATUS_IS = "आपका मास्टरनोड स्टेटस है" # Your masternode status is
+MN_STATE = "आपका मास्टरनोड <b>{state}</b> स्स्टेट में है" # Your masternode is in <b>{state}</b> state
+MN_BAD_IP = "IP एड्रेस अमान्य है!" # The IP address is invalid!
+MN_BAD_PRIVKEY = "प्राइवेट की अमान्य है" # The private key is invalid
+MN_NOT_ENOUGH_COLLAT = "आपको एक मास्टरनोड बनाने के लिए <b>{amount} और {ticker}</b> की आवश्यकता है!" # You need <b>{amount} more {ticker}</b> to create a Masternode!
+MN_ENOUGH_BUT_NO_COLLAT = "आपके पास मास्टरनोड के लिए पर्याप्त बैलेंस है, लेकिन {amount} {ticker} का कोई मान्य कोलेटरल UTXO नहीं है" # You have enough balance for a Masternode, but no valid collateral UTXO of {amount} {ticker}
+MN_COLLAT_NOT_SUITABLE = "यह मास्टरनोड के लिए उपयुक्त UTXO नहीं है" # This is not a suitable UTXO for a Masternode
+MN_CANT_CONNECT = "RPC नोड से कनेक्ट नहीं हो पा रहा है!" # Unable to connect to RPC node!
+CONTACTS_ENCRYPT_FIRST = "आपको संपर्कों का उपयोग करने से पहले "{button}" पर क्लिक करना होगा!" # You need to hit "{button}" before you can use Contacts!
```
Comment (Duddino) [inline locale/hi/translation.toml:279]:
```suggestion
CONTACTS_ENCRYPT_FIRST = "आपको संपर्कों का उपयोग करने से पहले \"{button}\" पर क्लिक करना होगा!" # You need to hit "{button}" before you can use Contacts!
```

File: locale/hi/translation.toml [hunk 6]
```diff
@@ -0,0 +1,335 @@
...
+CONTACTS_NAME_REQUIRED = "नाम आवश्यक है!" # A name is required!
+CONTACTS_NAME_TOO_LONG = "वह नाम बहुत लंबा है!" # That name is too long!
+CONTACTS_CANNOT_ADD_YOURSELF = "आप खुद को संपर्क के रूप में जोड़ नहीं सकते!" # You cannot add yourself as a Contact!
+CONTACTS_ALREADY_EXISTS = "<b>संपर्क पहले से मौजूद है!</b><br>आपने पहले ही इस संपर्क को सेव किया है" # <b>Contact already exists!</b><br>You already saved this contact
+CONTACTS_NAME_ALREADY_EXISTS = "<b>संपर्क का नाम पहले से मौजूद है!</b><br>यह संभावित रूप से एक फ़िशिंग प्रयास हो सकता है, सतर्क रहें!" # <b>Contact name already exists!</b><br>This could potentially be a phishing attempt, beware!
+CONTACTS_EDIT_NAME_ALREADY_EXISTS = "<b>संपर्क पहले से मौजूद है!</b><br>एक संपर्क पहले से ही "{strNewName}" के नाम से है!" # <b>Contact already exists!</b><br>A contact is already called "{strNewName}"!
```
Comment (Duddino) [inline locale/hi/translation.toml:285]:
```suggestion
CONTACTS_EDIT_NAME_ALREADY_EXISTS = "<b>संपर्क पहले से मौजूद है!</b><br>एक संपर्क पहले से ही \"{strNewName}\" के नाम से है!" # <b>Contact already exists!</b><br>A contact is already called "{strNewName}"!
```

File: locale/hi/translation.toml [hunk 7]
```diff
@@ -0,0 +1,335 @@
...
+MIGRATION_ACCOUNT_FAILURE = "आपका खाता पुनर्प्राप्त करने में विफल। कृपया इसे रिइम्पोर्ट करें" # Failed to recover your account. Please reimport it.
+APP_INSTALLED = "ऐप स्थापित हो गया!" # App Installed!
+WALLET_FIREFOX_UNSUPPORTED = "<b>फायरफॉक्स इसका समर्थन नहीं करता!</b><br>दुर्भाग्यवश, फायरफॉक्स हार्डवेयर वॉलेट का समर्थन नहीं करता है" # <b>Firefox doesn't support this!</b><br>Unfortunately, Firefox does not support hardware wallets
+CONFIRM_POPUP_DELETE_ACCOUNT = "यह आपके सभी डेटा को हटा देगा, जिसमें मास्टरनोड्स, संपर्क और प्राइवेट की शामिल हैं!" # This will delete all your data, including masternodes contacts and private keys!
+CONFIRM_POPUP_DELETE_ACCOUNT_TITLE = "क्या आप सुनिश्चित हैं?" # Are you sure?
+CONFIRM_LEDGER_TX = "कन्फर्म करें कि यह ट्रांजैक्शन आपके {hardwareWallet} पर जोड़े गए ट्रांजैक्शन से मेल खाता है" # Confirm this transaction matches the one on your {hardwareWallet}
+CONFIRM_LEDGER_TX_OUT = "आप {value} {ticker} को <div class="inline-address">{address}</div> भेजेंगे" # You will send {value} {ticker} to <div class="inline-address">{address}</div>
```
Comment (Duddino) [inline locale/hi/translation.toml:333]:
```suggestion
CONFIRM_LEDGER_TX_OUT = "आप {value} {ticker} को <div class=\"inline-address\">{address}</div> भेजेंगे" # You will send {value} {ticker} to <div class="inline-address">{address}</div>
```

**Review by Duddino — CHANGES_REQUESTED** [review]
Need to escape some quotes

**Review by Duddino — CHANGES_REQUESTED** [review]
You need to add the language to `arrActiveLangs` in `i18n.js`

**Review by JSKitty — APPROVED** [review]
**tACK** 4e4ac2a50ecd7d5015f56387072228f8abbc5d56

We'll update these translations for any new strings added and/or changed prior to v2.0 launch.

**Pull request merged**